### PR TITLE
Update label name for dotnet-docs and dotnet-api-docs

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: _data/projects/dotnet-api-docs.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ tags:
 - ".net"
 - documentation
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/dotnet/dotnet-api-docs/labels/up-for-grabs
+  name: help wanted
+  link: https://github.com/dotnet/dotnet-api-docs/labels/help wanted
```
Comment (mairaw) [inline _data/projects/dotnet-api-docs.yml]:
```suggestion
  link: https://github.com/dotnet/dotnet-api-docs/labels/help%20wanted
```

File: _data/projects/dotnet-docs.yml [hunk 2]
```diff
@@ -6,8 +6,8 @@ tags:
 - ".net"
 - documentation
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/dotnet/docs/labels/up-for-grabs
+  name: help wanted
+  link: https://github.com/dotnet/docs/labels/help wanted
```
Comment (mairaw) [inline _data/projects/dotnet-docs.yml]:
```suggestion
  link: https://github.com/dotnet/docs/labels/help%20wanted
```

**Comment by gewarren** [comment]
Thanks @mairaw, you rock!

**Comment by mairaw** [comment]
It's still not happy 😢

File: _data/projects/dotnet-api-docs.yml [hunk 3]
```diff
@@ -6,8 +6,8 @@ tags:
 - ".net"
 - documentation
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/dotnet/dotnet-api-docs/labels/up-for-grabs
+  name: help wanted
```
Comment (gewarren) [inline _data/projects/dotnet-api-docs.yml:9]:
```suggestion
  name: help%20wanted
```

File: _data/projects/dotnet-docs.yml [hunk 4]
```diff
@@ -6,8 +6,8 @@ tags:
 - ".net"
 - documentation
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/dotnet/docs/labels/up-for-grabs
+  name: help wanted
```
Comment (gewarren) [inline _data/projects/dotnet-docs.yml:9]:
```suggestion
  name: help%20wanted
```

File: _data/projects/dotnet-api-docs.yml [hunk 5]
```diff
@@ -6,8 +6,8 @@ tags:
 - ".net"
 - documentation
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/dotnet/dotnet-api-docs/labels/up-for-grabs
+  name: help%20wanted
```
Comment (gewarren) [inline _data/projects/dotnet-api-docs.yml]:
```suggestion
  name: "help wanted"
```

File: _data/projects/dotnet-docs.yml [hunk 6]
```diff
@@ -6,8 +6,8 @@ tags:
 - ".net"
 - documentation
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/dotnet/docs/labels/up-for-grabs
+  name: help%20wanted
```
Comment (gewarren) [inline _data/projects/dotnet-docs.yml]:
```suggestion
  name: "help wanted"
```

File: _data/projects/dotnet-api-docs.yml [hunk 7]
```diff
@@ -6,8 +6,8 @@ tags:
 - ".net"
 - documentation
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/dotnet/dotnet-api-docs/labels/up-for-grabs
+  name: "help wanted"
```
Comment (gewarren) [inline _data/projects/dotnet-api-docs.yml]:
```suggestion
  name: help wanted
```

File: _data/projects/dotnet-api-docs.yml [hunk 8]
```diff
@@ -6,8 +6,8 @@ tags:
...
+  link: https://github.com/dotnet/dotnet-api-docs/labels/help%20wanted
```
Comment (gewarren) [inline _data/projects/dotnet-api-docs.yml:10]:
```suggestion
  link: https://github.com/dotnet/dotnet-api-docs/labels/help%20wanted
```

File: _data/projects/dotnet-docs.yml [hunk 9]
```diff
@@ -6,8 +6,8 @@ tags:
 - ".net"
 - documentation
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/dotnet/docs/labels/up-for-grabs
+  name: "help wanted"
```
Comment (gewarren) [inline _data/projects/dotnet-docs.yml]:
```suggestion
  name: help wanted
```

File: _data/projects/dotnet-docs.yml [hunk 10]
```diff
@@ -6,8 +6,8 @@ tags:
...
+  link: https://github.com/dotnet/docs/labels/help%20wanted
```
Comment (gewarren) [inline _data/projects/dotnet-docs.yml:10]:
```suggestion
  link: https://github.com/dotnet/docs/labels/help%20wanted
```

**Comment by gewarren** [comment]
@shiftkey This appears to be a bogus validation error. Can you help to merge this?

File: _data/projects/dotnet-api-docs.yml (hunk 7 above)
Comment (mairaw) [inline _data/projects/dotnet-api-docs.yml]:
```suggestion
  name: help wanted
```

File: _data/projects/dotnet-docs.yml (hunk 9 above)
Comment (mairaw) [inline _data/projects/dotnet-docs.yml]:
```suggestion
  name: help wanted
```

**Review by mairaw — COMMENTED** [review]
never mind, needed a refresh

**Comment by ritwik12** [comment]
@gewarren @mairaw Thanks for resolving the conflicts.

**Pull request merged**